e2e/test/Helpers/templates/FaultInjectionPoolingOverAmqp.cs

```diff
@@ -94,7 +94,7 @@ public static class FaultInjectionPoolingOverAmqp
                 watch.Start();
 
                 logger.Trace($"{nameof(FaultInjectionPoolingOverAmqp)}: {testDevices[0].Id} Requesting fault injection type={faultType} reason={reason}, delay={delayInSec}s, duration={durationInSec}s");
-                var faultInjectionMessage = FaultInjection.ComposeErrorInjectionProperties(faultType, reason, delayInSec, durationInSec);
+                using Client.Message faultInjectionMessage = FaultInjection.ComposeErrorInjectionProperties(faultType, reason, delayInSec, durationInSec);
```

Comment:
Why did you make the type explicit?

Reply:
This is because the returned type isn't obvious based on the method call. While we prefer to use `var` only when the type is obvious, eg. when doing a `new <ClassName>()`, we have deviations from that in a lot of places in our code. My approach is to fix it as I go.
This change isn't related to adding NET 5.0 support.